src/components/TasksTable/index.jsx

```diff
@@ -92,33 +104,54 @@ export default class TasksTable extends Component {
     });
   };
 
+  handleChipClick = tag => event => {
+    const query = this.getQuery();
+    const newQuery =
+      query.tag === tag ? omit(['tag'], query) : { ...query, tag };
+
+    this.props.history.push({
+      search: `?${stringify(newQuery)}`,
+    });
+
+    event.preventDefault();
+  };
+
   render() {
     const { items, classes } = this.props;
-    const { sortBy, sortDirection } = this.getQuery();
-    const data = this.getTableData(sortBy, sortDirection, items);
+    const iconSize = 14;
+    const query = this.getQuery();
+    const { sortBy, sortDirection, tag } = query;
+    const data = this.getTableData(sortBy, sortDirection, tag, items);
 
     return (
       <div className={classes.tableWrapper}>
         <DataTable
           title="Bugs & Issues"
           items={data}
           renderRow={item => (
-            <TableRow
-              hover
-              target="_blank"
-              rel="noopener noreferrer"
-              tabIndex={-1}
-              key={item.summary}
-              component="a"
-              href={item.url}
-              className={classes.link}>
+            <TableRow hover tabIndex={-1} key={item.summary}>
               <TableCell component="th" scope="row">
                 {item.project}
               </TableCell>
-              <TableCell className={classes.summary}>{item.summary}</TableCell>
+              <TableCell
+                className={classes.summary}
+                target="_blank"
+                rel="noopener noreferrer"
+                component="a"
+                href={item.url}>
+                {item.summary}
+                <LinkIcon size={iconSize} className={classes.icon} />
+              </TableCell>
               <TableCell className={classes.tags}>
                 {item.tags.map(tag => (
-                  <Chip key={tag} label={tag} className={classes.chip} />
+                  <Chip
+                    key={tag}
+                    label={tag}
+                    className={classNames({
+                      [classes.clickedChip]: tag === query.tag,
+                    })}
+                    onClick={this.handleChipClick(tag)}
+                  />
```

Comment:
Hovering over the chip makes the chip grey with white text. Going from green-blue to grey on hover is not a smooth transition.In addition, gray on white text is not enough contrast making it hard for the eye to read the text. Could we instead just use a different shade of the green-blue color? You can try exporting the `emphasize` function from the library and provide it with the current chip color.

To export the `emphasize` function, you will do `import { emphasize } from '@material-ui/core/styles/colorManipulator';`

https://github.com/mui-org/material-ui/blob/8bc7d11c9a445de5acbf0de8ec550e3bdd23c565/packages/material-ui/src/Chip/Chip.js#L42-L44